DOCS/man/options.rst

```diff
@@ -2906,11 +2906,6 @@ Input
 
     (This option was renamed from ``--input-x11-keyboard``.)
 
-``--input-app-events=<yes|no>``
-    (OS X only)
-    Enable/disable application wide keyboard events so that keyboard shortcuts
-    can be processed without a window. Enabled by default (except for libmpv).
-
```

Comment:
yeah, don't think we need a deprecation. this shouldn't be used anyway other than for testing, in the libmpv case. i also doubt that anyone explicitly deactivated this. otherwise someone would have reported the not working key input in fullscreen.